packages/react/src/primitives/SearchField/__tests__/SearchField.test.tsx

```diff
@@ -70,23 +70,21 @@ describe('SearchField component', () => {
   });
 
   it('should forward callback ref to DOM element', async () => {
-    let ref: HTMLInputElement;
-
-    const setRef = (node) => (ref = node);
+    const ref = React.createRef<HTMLInputElement>();
```

Comment:
I think using the callback ref is intentional here if you look at the test name

Reply:
The `createRef` API was added in v16.3. Callback refs are no longer recommended in React, and while I am all for testing for edge cases, testing for functionality that has been considered legacy since versions of React below our supported version feels extraneous, not to mention potentially problematic if React was to fully drop support for callback refs